site/app/controllers/admin/ReportController.php

```diff
@@ -367,6 +381,23 @@ private function saveUserToFile(string $base_path, User $user, array $ggs, LateD
         file_put_contents(FileUtils::joinPaths($base_path, $user->getId() . '_summary.json'), FileUtils::encodeJson($user_data));
     }
 
+    /**
+     * Generates a summary of all polls over a semester if polling is enabled
+     * @param string $base_path the base path to store the report
+     */
+    private function generatePollSummaryInternal(string $base_path): void {
+        $polls = $this->core->getQueries()->getPolls();
+        $polls_data = [];
+        foreach ($polls as $poll) {
+            $polls_data[] = [
+                "id" => $poll->getID(),
+                "responses" => $poll->getUserResponses()
+            ];
+        }
+        file_put_contents(FileUtils::joinPaths($base_path, "poll_responses_summary.json"), FileUtils::encodeJson($polls_data));
+        file_put_contents(FileUtils::joinPaths($base_path, "poll_data_summary.json"), FileUtils::encodeJson(PollUtils::getPollExportData($polls)));
```

Comment:
Use [`FileUtils::writeJsonFile`](https://github.com/Submitty/Submitty/blob/a9e09405eaf6b370fdd8978805a144a71042306e/site/app/libraries/FileUtils.php#L287):

```suggestion
        FileUtils::writeJsonFile(FileUtils::joinPaths($base_path, "poll_responses_summary.json"), $polls_data);
        FileUtils::writeJsonFile(FileUtils::joinPaths($base_path, "poll_data_summary.json"), PollUtils::getPollExportData($polls));
```